src/esp/metadata/attributes/ObjectAttributes.h

```diff
@@ -89,6 +89,12 @@ class AbstractObjectAttributes : public AbstractAttributes {
    */
   double getUnitsToMeters() const { return get<double>("units_to_meters"); }
 
+  /**
+   * @brief If not visible can add dynamic non-rendered object into a scene
+   * object.  If is not visible then should not add object to drawables.
+   */
+  void setIsVisible(bool isVisible) { set("is_visible", isVisible); }
```

Comment:
Why is_visible instead of "visibility"?

Reply:
because that is the tag in the json we use.